### DISCUSSION TITLE
Scroll single item with long width?

### DISCUSSION THREAD

**Discussion by georgienwankwo** [asmyshlyaev177/react-horizontal-scrolling-menu#238] [Q&A]
I can't implement this. Can anyone help?

https://codesandbox.io/s/basic-example-forked-i5zuzw?file=/src/card.tsx

**Comment by AlexSmyshlyaev-sm** [comment]
Not sure this library is good choice for the task.
Can detect container width by `scrollWidth` on scroll container and update `scrollLeft`.
https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft

**Comment by georgienwankwo** [comment]
Alright, thanks!
